### ISSUE TITLE
Can't sign in mobile devices (page not responsive)

### ISSUE THREAD

**Issue by tasagore** [LibreSign/libresign#1308]
Hi

I've tried to sign a document with two mobile devices (iOS and Android). The first one show the document, but nothing else, in the first load I can see a small part of the generate password button, I can generate if but after that I can't see any other button, doesn't matter if I try in portrait mode, it's impossible to see the buttons. The second one (Android) doesn't show the document, rest of behavior same as iOS, can't see the buttons and the page can't be resized/moved.

Does anybody try to sign with mobile devices with the latest v6.2.2?

The validation page has also problems that I don't remember to have in previous, I can't see the "view document" button.

**Comment by vitormattos** [comment]
Duplicated of #1263

**Comment by tasagore** [comment]
Not the same thing. I can’t see the whole page  no matter if iOS or Android so I can’t sign the document. #1263 seems to be different.

**Comment by tasagore** [comment]
This is an example of what happens when you try to sign with a mobile device. If you load the signature page in a desktop browser like chrome it appears like this, the PDF at the left, the signature icon at the right:
<img width="1232" alt="image" src="https://user-images.githubusercontent.com/54774469/211631740-36936d4c-04b9-44f7-bac1-8ac7f4404ef3.png">

But if you preview it for example in iPhone XR mode, this is what happens:

![image](https://user-images.githubusercontent.com/54774469/211632242-96c62152-2a21-4070-9084-40e012a3817c.png)

The problem is that is impossible to show the PDF preview and the signature button on two columns in a mobile device (there is no room for that). In that case should be under the PDF, not at the right. This is how NC shows a PDF in a mobile:

<img width="966" alt="image" src="https://user-images.githubusercontent.com/54774469/211633609-ece2a32e-d18a-4409-906a-cb490431db5d.png">

No way to have other column for the signature button. Tested in a few Android and iOS devices, and also in desktop browser with mobile profile, no one can show the page so it's impossible to sign any doc in a mobile device.

I've tried to find the page to modify the CSS but I didn't find it, will try to look again tomorrow and if I find it I'll try to make some css mods to show the button under the PDF and see if it works.
